ember/app/ui/components/identity-emails/template.hbs

```diff
@@ -0,0 +1,112 @@
+<div
+  {{did-insert this.onUpdate}}
+  {{did-update this.onUpdate @identity}}
```

Comment:
Please consider the feedback on the previous PR: https://github.com/adfinis-sygroup/mySAGW/pull/50#discussion_r572771370

Reply:
I looked [how it was done in ember-alexandria](https://github.com/projectcaluma/ember-alexandria/blob/main/addon/components/document-grid.hbs#L34) as I previously tried to do all my queries inside the `model` hooks of the routes.

And from what I've seen during development, there are no multiple requests. It all handles as expected.

The problem with the `setupController` hooks is that this only works for controllers. Many of these `did-insert`/`did-update` (including this one) are located in components.

For me, this is a non-issue with no clear solution.

Reply:
Here's the [commit](https://github.com/projectcaluma/ember-alexandria/commit/5a29cf4952ff88c20a863dc1ca7f57fcf2de7759). Not sure who did the review.

Reply:
It seems there was no review :see_no_evil: @anehx what do you say?

Reply:
As already stated in https://github.com/adfinis-sygroup/mySAGW/pull/50#discussion_r572771370 I don't know what the best way to go is. If you're using QPs you'll obviously need to do it via the routes `setupController` hook, if not, this is fine. I won't tell you what to do, just keep it consistent.

Reply:
So, what is the verdict @czosel?

Reply:
Keep it 🙃